### PR TITLE
Add example bundle

### PR DESCRIPTION
**Pull request by vdice**
Adds an example bundle around creation of a Packer-built Azure VM image and an Azure VM from this image.

### CODE REVIEW

File: examples/packer-azure-windows/windows.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    {
+      "type": "powershell",
+      "elevated_user": "packer",
+      "elevated_password": "{{.WinRMPassword}}",
+      "inline": [
+        "# Turn off PowerShell execution policy restrictions",
+        "Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope LocalMachine",
+        "# Install Chocolatey",
+        "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))",
+        "# Install Docker Version Manager",
```
Comment (vdice) [inline examples/packer-azure-windows/windows.json:51]: @carolynvs thank you for the suggestion to try dvm!  Indeed, installation appears to go great on packer build and the azure vm is successfully created, but I'm currently greeted with a black screen when RDP-ing into the subsequent VM.  I'm sure this isn't due to dvm, but just wanted to update on progress.  Exciting!

**Comment by carolynvs** [comment]
Woooooo! Ooo! Oo! I’m pretty happy to learn that dvm is still solving problems. 💅

Not sure about the blank screen. Maybe you need to install the desktop module? Is that still a thing on windows server?

**Comment by vdice** [comment]
@carolynvs I tore down and re-installed, this time adding an open port 80 block, as instructed in https://docs.microsoft.com/en-us/azure/virtual-machines/windows/build-image-with-packer#create-a-vm-from-the-packer-image.  This time, RDP-ing in worked (GUI/Desktop shows, can interact.) However, I don't quite see why the port open would do this... could have just been a glitch in the matrix on the first run.

Alas, the userspace in the resulting VM doesn't appear to have `dvm` or `docker` around, so I need to figure why this is... (other installed deps like `git`, `go` and `make` are ready to go -- thanks Chocolatey!) 

I may merge this as is and put the continuing Docker on a Windows VM Saga on hold ... but we've made it very far!

**Comment by carolynvs** [comment]
I think it's a good idea to merge incrementally. Looks like there is a huge chunk of work here.

File: examples/packer-azure-windows/Dockerfile.tmpl [hunk 2]
```diff
@@ -0,0 +1,21 @@
+FROM quay.io/deis/lightweight-docker-go:v0.2.0
```
Comment (carolynvs) [inline examples/packer-azure-windows/Dockerfile.tmpl]: nit: I vote for not using this as the base docker image as people external to our team are not familiar with it. If we can use maintained / stock base images, that would be preferred.

In Porter we have stopped using that base image and switched to 

```
FROM debian:stretch

ARG BUNDLE_DIR

RUN apt-get update && apt-get install -y ca-certificates
```

Argh! I just realized that I switched us over for the default template but not the `Dockerfile.tmpl`. I'm going to submit a PR to fix that now. 😊

**Review by carolynvs — APPROVED** [review]
LGTM

File: examples/packer-azure-windows/Dockerfile.tmpl (hunk 2 above)
Comment (vdice) [reply examples/packer-azure-windows/Dockerfile.tmpl]: Ah, thanks for pointing this out.  That stock template isn't needed at this time, so I've just removed it.

**Pull request merged**